contentcuration/contentcuration/models.py

```diff
@@ -1867,6 +1893,9 @@ def save(self, skip_lock=False, *args, **kwargs):
             if changed_ids:
                 ContentNode.objects.filter(id__in=changed_ids).update(changed=True)
 
+        if is_create:
+            ContentNode.filter_by_pk(pk=self.id).update(title_description_search_vector=POSTGRES_SEARCH_VECTOR)
```

Comment:
Is there a strong need for this? We are setting during publish, and no newly created node is going to be in a published channel until the publish happens.

Reply:
We strongly need this because users can search by their own "editable" channels which won't be in published state so to index those editable channels we are doing that operation.

Reply:
@vkWeb I don't think importing content from a user's own unpublished channel is a primary use case for importing resources, and therefore isn't a strong need here. Technically, they can do this today, but we've already discussed that dropping that use case does make sense since publishing ensures that the resource can be published and the act of publishing itself is a sort of final approval on the state of the resource.

Reply:
oh wow, if we are removing un-published search use case then triggers won't be required in any of the tables! :partying_face: